### PR TITLE
Added Twitter Bot Blog Post

### PR DESCRIPTION
**Pull request by techmaharaj**
Added new blog post for developing serverless Twitter Bot on Fission using Python.

### CODE REVIEW

File: content/en/blog/serverless-twitter-bot-fission/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+{{< figure src="/images/featured/serverless-twitterbot-fission.png" alt="Developing Serverless Twitter Bot on Fission" height="600" width="1000">}}
+
+## Understanding Serverless The Twitter Bot
+
+Before I take you through the code, let me help you understand what our serverless Twitter Bot will do. Below are a few pointers to help you understand the app and the use case better:
+
+* Respond to any tweet that mentions you with a generic message
+* Send a notification to Slack channel about the mention
+* Will be deployed as Fission function
+* Will execute at regular intervals using Fission’s Time Trigger
```
Comment (sanketsudake) [inline content/en/blog/serverless-twitter-bot-fission/index.md:24]: Should we link time trigger page here ?

File: content/en/blog/serverless-twitter-bot-fission/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+The serverless Twitter bot is developed using Python and uses the following libraries:
```
Comment (sanketsudake) [inline content/en/blog/serverless-twitter-bot-fission/index.md:26]: developed in Python language

File: content/en/blog/serverless-twitter-bot-fission/index.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+## Deploying Twitter Bot to Fission
+
+Create a Python environment
+
+```bash
+fission environment create --name python --image fission/python-env --builder fission/python-builder:latest
+```
+
+Create a zip archive as sample.zip archive by executing package.sh script
```
Comment (sanketsudake) [inline content/en/blog/serverless-twitter-bot-fission/index.md:124]: There is no mention of source from where it should be taken. 
Only mention of package.sh directly.

File: content/en/blog/serverless-twitter-bot-fission/index.md (hunk 1 above)
Comment (techmaharaj) [reply content/en/blog/serverless-twitter-bot-fission/index.md]: Linked.

File: content/en/blog/serverless-twitter-bot-fission/index.md (hunk 3 above)
Comment (techmaharaj) [reply content/en/blog/serverless-twitter-bot-fission/index.md:124]: The package.sh file is in the source code folder itself, hence it if is executed, all the content will be added to archive.

Comment (sanketsudake) [reply content/en/blog/serverless-twitter-bot-fission/index.md:124]: can you please mention source code repo and path before execution command ? 
I couldn't infer it from blog.

Comment (sanketsudake) [reply content/en/blog/serverless-twitter-bot-fission/index.md:124]: Ok I see its mentioned at top . 

image

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**